tests/js/spec/views/settings/settingsIndex.spec.jsx

```diff
@@ -10,20 +10,32 @@ describe('SettingsIndex', function() {
   let wrapper;
 
   it('renders', function() {
-    wrapper = shallow(<SettingsIndex organization={TestStubs.Organization()} />);
+    wrapper = shallow(
+      <SettingsIndex
+        router={TestStubs.router()}
+        organization={TestStubs.Organization()}
+      />
```

Comment:
Are there any functional tests we could add for the new behavior? Should we have an acceptance test with a visual snapshot that covers the new menu?

Reply:
Added 👍